kytos/core/kytosd.py

```diff
@@ -145,7 +145,7 @@ async def start_shell_async():
 
     loop.call_soon(controller.start)
 
-    if controller.options.foreground:
+    if controller.options.interactive:
```

Comment:
We don't need to change our default behavior here.

Using this patch, We are going to need an extra argument (`-i`) to run in interactive mode. This is changing how kytosd run and how a lot of scripts and users are expecting to run. Our documentation, tutorials everything is assuming this.

Also, I'm still not sure if we really need this extra parameter. There is no such option in `podman` to handle this ? Because, right now kytos is starting exactly as a POSIX daemon. Like, apache, mysql, and many other daemons.

Unless I'm missing something, I don't see a reason to change the default beahviour in kytos.

Let's try to reduce the problem to a podman or docker. (let's remove the upper layers that we have with kubernetes). Can you reproduce the problem only with docker/podman? There is any option inside podman that will proper handle this?